cli/command/system/info.go

```diff
@@ -329,6 +333,16 @@ func printStorageDriverWarnings(dockerCli command.Cli, info types.Info) {
 	}
 }
 
+func printOrchestratorInfos(dockerCli command.Cli) error {
+	orchestrator, err := command.GetStackOrchestrator("", dockerCli.ConfigFile().StackOrchestrator, dockerCli.Err())
```

Comment:
Instead of requesting this information here; should we extend the `info` type to allow adding local information to be added by the CLI?